README.md

```diff
@@ -1,15 +1,65 @@
 # hc-web-client
 
-Thin wrapper around `rpc-websockets` to enable calling zome functions in Holochain apps installed in a container.
+Thin wrapper around `rpc-websockets` to enable calling zome functions in Holochain apps installed in a conductor. This module aims to fill a similar role to web3.js and allow for connecting to a holochain instance in a variety of scenarios.
+
+## Installation
+
+The module can either be imported and used as part of a compiled web application or used in pure HTML. In the first case it is reccomended to download and install via npm `npm install @holochain/hc-web-client`. It can then be imported or required using
+
+```javascript
+const { connect } = require('@holochain/hc-web-client')
+// or
+import { connect } from '@holochain/hc-web-client'
+```
+
+For the pure HTML case the index.js file in the `dist/` folder must be moved to your project directory and imported using
+```html
+<script type="text/javascript" src="path/to/holoclient.js"></script>
+```
 
 ## Usage
 
-The holochain container exposes methods for each function of each zome of each running instance. It also exposes a method, `info/instances` to get information about all instances. Your app should typically use this first to get instance info, so that your UI can only be concerned with DNA hashes (and perhaps agent IDs) rather than instance IDs.
```

Comment:
why remove this? could just leave it as a note afterwards?

Reply:
This info is kind of out of date but I added a paragraph describing how to use the client to connect to the admin RPC